### PR TITLE
Issue #84: Update UI to use segmented bar

### PR DESCRIPTION
**Pull request by mjshepherd**

### CODE REVIEW

File: ui/src/main/resources/PhenoTips/SimilarCases.xml [hunk 1]
```diff
@@ -1558,7 +1530,6 @@
 }
 .match-details .variants-data .segmented-bar .segmented-unit-fill {
```
Comment (sdumitriu) [inline ui/src/main/resources/PhenoTips/SimilarCases.xml]: Shouldn't all the `.segmented*` rules be removed? The segmented bar widget is supposed to be fully usable without any other style rules except setting the font size and color.

Comment (mjshepherd) [inline ui/src/main/resources/PhenoTips/SimilarCases.xml]: The widget is usable without external css but css may still be applied to it. Float for example could either be applied in javascript or in css.

**Comment by buske** [comment]
@mjshepherd please rebase

**Comment by buske** [comment]
Incorporated into d623306ffc93efcaa889d283399cbc6c1e819048.

**Pull request closed without merging**